howso/attribute_maps.amlg

```diff
@@ -324,6 +324,62 @@
 				)
 			)
 
+			(if (> (call !GetNumTrainingCases) 0)
+				(seq
+					(if (contains_index bounds_map "min")
+						(if
+							(<
+								(retrieve_from_entity
+									(first (contained_entities (list
```

Comment:
change this code and the one below to be:
(first (contained_entities
    (list (query_exists feature) (query_min feature 1))
))